### PR TITLE
Adds UV click event listener and sends custom event to Plausible

### PR DESCRIPTION
**Pull request by sdellis**
This may be a naive approach, but it listens for a click event on the #uv element, logs it (for testing), and dispatches a `uv-click` event to Plausible. I am unsure of how to test this in Plausible to make sure it was received.

### CODE REVIEW

File: app/javascript/viewer/uv_manager.js [hunk 1]
```diff
@@ -137,6 +138,26 @@ export default class UVManager {
...
+    const uvClick = new Event("uv-click", { bubbles: true });
+
+    uvElement.addEventListener(
+      "click",
+      (event) => {
+        /* … */
+        uvElement.dispatchEvent(uvClick)
+        const cachedOpen = window.open
+        window.open = function (url, target, features) {
+          window.plausible('uv-click', { props: { url } })
```
Comment (tpendragon) [inline app/javascript/viewer/uv_manager.js:152]: I don't think you need to override `window.open`, just put this below 149 and delete all this open stuff.

For the download controller that was more about attaching to the way UV does downloads.

File: app/javascript/viewer/uv_manager.js [hunk 2]
```diff
@@ -137,6 +138,26 @@ export default class UVManager {
     }
   }
 
+  bindUvClick () {
+    const uvElement = document.getElementById('uv')
+    const uvClick = new Event("uv-click", { bubbles: true });
+
+    uvElement.addEventListener(
```
Comment (tpendragon) [inline app/javascript/viewer/uv_manager.js:145]: What are the chances we can get a test? I think I wrote one for plausible stuff in the viewer test before that you could use as a model.

**Review by tpendragon — CHANGES_REQUESTED** [review]

File: app/javascript/viewer/uv_manager.js (hunk 2 above)
Comment (sdellis) [reply app/javascript/viewer/uv_manager.js:145]: Done!

**Review by tpendragon — APPROVED** [review]

**Comment by sdellis** [comment]
Closes #6438

**Pull request merged**